modules/op-release-notes-1-8.adoc

```diff
@@ -12,13 +12,145 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// https://github.com/tektoncd/pipeline/pull/4887
+
+* Implement Parameter Propagation. With this update, parameters in embedded specifications are propagated without mutations.
+//https://github.com/tektoncd/pipeline/pull/4845
+
+* With this update, you can specify the required metadata of a `Task` resource referenced by a `PipelineRun` resource by using annotations and labels. This way, `Task` metadata that depends on the execution context is available during the pipeline run.
+// https://github.com/tektoncd/pipeline/pull/4834
+
+* This update adds support for object or dictionary types in `params`  and `results` values. This change affects backward compatibilities and sometimes breaks forward compatibility, such as using an earlier client with a later {pipelines-title} version.
+Action required: This update changes the `ArrayOrStruct` structure, which affects the project using the go API as a library.
+// TBD what action?
+// https://github.com/tektoncd/pipeline/pull/4786
+
+* This update adds a `SkippingReason` value to the `SkippedTasks` field of the `PipelineRun` status fields so that users know why a given PipelineTask was skipped.
+// https://github.com/tektoncd/pipeline/pull/4829
+
+* As an alpha feature, this update supports using an `array` type for emitting results from a `Task` object. The type of the result is changed from `string` to `ArrayOrString`.
++
+A task can specify a type to produce an array result, such as:
++
+[source,yaml]
+----
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: write-array
+  annotations:
+    description: |
+      A simple task that writes array
+spec:
+  results:
+    - name: array-results
+      type: array
+      description: The array results
+...
+----
++
+And the task script can populate the result in an array form with:
++
+[source,terminal]
+----
+$ echo -n "[\"hello\",\"world\"]" | tee $(results.array-results.path)
+----
++
+This feature is part of the TEP-0076 and is in progress to index into the array result while consuming that result.
+// https://github.com/tektoncd/pipeline/pull/4818
+
+=== Triggers
+
+* This update transitions the `TriggerGroups` field in the `EventListener` specification from being an alpha feature to being a stable feature. Using this field, you can specify a set of interceptors before selecting and running a group of triggers. It is available by default, and you do not need to change the configuration to use it.
+// https://github.com/tektoncd/triggers/pull/1345(#1345)
+// @dlorenc
+
+* With this update, the `Trigger` resource supports end-to-end secure connections by running the `ClusterInterceptor` server using HTTPS.
+// https://github.com/tektoncd/pipeline/pull/1333(#1333)
+// @savitaashture
```

Comment:
@savitaashture Please review and approve. Thanks.

Reply:
Please ignore the fact that GitHub closed this PR and leave your review here anyway.